process_manager/templates/process_manager/partials/process_table.html

```diff
@@ -1,2 +1,46 @@
+{% extends "django_tables2/table.html" %}
 {% load render_table from django_tables2 %}
+{% load querystring from django_tables2 %}
+{% block table.thead %}
+  <div>
+    <button style="padding: 5px 12px;
+                   background-color: #f8f9fa;
+                   color: #495057;
+                   border: 1px solid #ddd;
+                   border-radius: 8px;
+                   font-size: 0.9em;
+                   font-weight: 600;
+                   cursor: pointer;
+                   transition: background-color 0.3s ease"
+            onmouseover="this.style.backgroundColor='#e2e6ea'"
+            onmouseout="this.style.backgroundColor='#f8f9fa'"
+            hx-get="{% url 'process_manager:process_table' %}?sort="
+            hx-target="div.table-container"
+            onclick="document.getElementById('sort-field').value = ''; document.getElementById('sort-param-display').textContent = 'None';">
+      Clear Sorting
+    </button>
+  </div>
+  {% if table.show_header %}
+    <thead class="custom-thead" {{ table.attrs.thead.as_html }}>
+      <tr>
+        {% for column in table.columns %}
+          <th class="custom-th" {{ column.attrs.th.as_html }}>
+            {% if column.orderable %}
+              <a hx-get="{% url 'process_manager:process_table' %}?sort="
+                 hx-target="div.table-container"
+                 href="#"
+                 onclick=" let sortField = document.getElementById('sort-field'); let currentSort = sortField.value; let newSort = currentSort === '{{ column.name }}' ? '-{{ column.name }}' : '{{ column.name }}'; sortField.value = newSort; this.setAttribute('hx-get', `{% url 'process_manager:process_table' %}?sort=${newSort}`); "
```

Comment:
And this one would look:
```suggestion
                 _="let currentSort = #sort-field's value; set #sort-field's value to (if currentSort is '{{ column.name }}' then '-{{ column.name }}' else '{{ column.name }}') and set this's hx-get to '{% url "process_manager:process_table" %}?sort=' + #sort-field's value"
```

I've used Copilot to translate from JS to HS, so don't accept this blindly, but I think it looks sensible.

Reply:
I don't think that looks like a very valid translation unfortunately.
```suggestion
                 onclick=" let sortField = document.getElementById('sort-field'); let currentSort = sortField.value; let newSort = currentSort === '{{ column.name }}' ? '-{{ column.name }}' : '{{ column.name }}'; sortField.value = newSort; this.setAttribute('hx-get', `{% url 'process_manager:process_table' %}?sort=${newSort}`); "
```